### PR TITLE
TEP-0080 Example syntax with enclosing single/double quotes

### PR DESCRIPTION
**Pull request by skaegi**
This update add example syntax illustrating the use of enclosing single and double quotes for parameter names. I've used bracket notation to describe the syntax in place of subscript as that is in common use. Also, added an additional risk statement for `-` characters.

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
/cc @mattmoor

**Comment by vdemeester** [comment]
> Thanks for this @skaegi - we don't really have a process for updating a TEP already marked as "implemented". We could switch this back to "implementable" or open a new TEP instead. The latter feels like a bit too much for a small update though. @vdemeester @bobcatfish thoughts?

I would think : switch to implementable, get it in, get the update merged in the components, and mark it as implemented again.

**Review by mattmoor — APPROVED** [review]

**Comment by mattmoor** [comment]
Thanks for fixing my typos 😊

**Comment by jerop** [comment]
will lgtm after linting issue is resolved

**Comment by pritidesai** [comment]
hey @skaegi, thanks for updating the proposal, linting is failing with:

```
-|[TEP-0080](0080-support-domainscoped-parameterresult-names.md) | Support domain-scoped parameter/result names | implemented | 2021-08-19 |
+|[TEP-0080](0080-support-domainscoped-parameterresult-names.md) | Support domain-scoped parameter/result names | implementable | 2021-10-01 |
```

/approve

**Comment by skaegi** [comment]
hmm... I'm struggling with the lint check...
I'm getting...
```
diff --git a/teps/README.md b/teps/README.md
index b74cfc6..6624594 100644
--- a/teps/README.md
+++ b/teps/README.md
@@ -221,5 +221,5 @@ This is the complete list of Tekton teps:
 |[TEP-0071](0071-custom-task-sdk.md) | Custom Task SDK | proposed | 2021-06-15 |
 |[TEP-0072](0072-results-json-serialized-records.md) | Results: JSON Serialized Records | implementable | 2021-07-26 |
 |[TEP-0073](0073-simplify-metrics.md) | Simplify metrics | proposed | 2021-06-23 |
-|[TEP-0080](0080-support-domainscoped-parameterresult-names.md) | Support domain-scoped parameter/result names | implemented | 2021-08-19 |
+|[TEP-0080](0080-support-domainscoped-parameterresult-names.md) | Support domain-scoped parameter/result names | implementable | 2021-10-01 |
 |[TEP-0084](0084-endtoend-provenance-collection.md) | end-to-end provenance collection | proposed | 2021-09-16 |
```

I think this is because of "attempting" to change the date of something "implemented" to "implementable". I'm going to revert the TEP state back to how it was before and add a note to get this merged, but maybe the linter should be tweaked??

**Comment by jerop** [comment]
thanks @skaegi 😁 

/lgtm

**Pull request merged**
